lib/seahorn/BvOpSem2RawMemMgr.cc

```diff
@@ -399,34 +416,35 @@ PtrTy RawMemManager::ptrAdd(PtrTy ptr, Expr offset) const {
 /// Returns an expression describing the state of memory in \c memReadReg
 /// after the store
 /// \sa loadIntFromMem
-Expr RawMemManager::storeIntToMem(Expr _val, PtrTy ptr, MemValTy mem,
-                                  unsigned byteSz, uint64_t align) {
+RawMemManagerCore::MemValTy
+RawMemManagerCore::storeIntToMem(Expr _val, const PtrTy &ptr, MemValTy mem,
+                                 unsigned byteSz, uint64_t align) {
   Expr val = _val;
 
   unsigned offsetBits = getByteAlignmentBits();
-  bool wordAligned = offsetBits == 0 || align % wordSzInBytes() == 0;
+  bool wordAligned = offsetBits == 0 || align % wordSizeInBytes() == 0;
   if (!m_ignoreAlignment && !wordAligned) {
-    return storeUnalignedIntToMem(val, ptr, mem, byteSz);
+    return MemValTy(storeUnalignedIntToMem(val, ptr, mem, byteSz));
   }
 
   SmallVector<Expr, 16> words;
-  if (byteSz == wordSzInBytes()) {
+  if (byteSz == wordSizeInBytes()) {
     words.push_back(val);
-  } else if (byteSz < wordSzInBytes()) {
-    val = m_ctx.alu().doZext(val, wordSzInBits(), byteSz * 8);
+  } else if (byteSz < wordSizeInBytes()) {
+    val = m_ctx.alu().doZext(val, wordSizeInBits(), byteSz * 8);
     words.push_back(val);
   } else {
-    for (unsigned i = 0; i < byteSz; i += wordSzInBytes()) {
+    for (unsigned i = 0; i < byteSz; i += wordSizeInBytes()) {
       unsigned lowBit = i * 8;
-      Expr slice = bv::extract(lowBit + wordSzInBits() - 1, lowBit, val);
+      Expr slice = bv::extract(lowBit + wordSizeInBits() - 1, lowBit, val);
```

Comment:
use `alu().Extract` instead of `bv::extract`

Reply:
fixed.